payments/card-payments/index.md

```diff
@@ -27,29 +27,29 @@ TODO: This page needs serious clean-up.
                       header="Site under development"
                       body="This section of the Developer Portal is under construction and should not be used to integrate against Swedbank Pay's APIs yet." %}
 
-{% include jumbotron.html body="**Credit Card Payments** is the most popular,
+{% include jumbotron.html body="**Card Payments** is the most popular,
 versatile and global way to initate a transaction with a customer. Choose
 between our **Seamless View**, **Redirect**, or **Direct** integration options." %}
 
 * When properly set up in your merchant/webshop site and the payer starts the
```

Comment:
Can we introduce Card Payments with something else than a bullet list? The Jumbotron does a good job of setting the tone, but then I think we dive too fast into technical details.

Reply:
Seems like @trieutran91 has made a softer introduction in #77.